server/services/bookerService.ts

```diff
@@ -60,4 +60,9 @@ export default class BookerService {
       return { ...visitor, visitorDisplayId: index + 1, adult: isAdult(visitor.dateOfBirth) }
     })
   }
+
+  async getEligibleVisitors(bookerReference: string, prisonerNumber: string): Promise<Visitor[]> {
```

Comment:
This is the crux of this change: use 'eligible' visitors for booking journey - i.e. those with no restrictions